MCNPInput.cpp

```diff
@@ -49,6 +49,27 @@ static void strlower( std::string& str ){
   }
 }
 
+static std::string findMnemonic( std::string token1, std::string token2 ){
+  std::string mnemonic;
+  if(token1.find_first_of("1234567890-") != 0){
+    mnemonic = token1;
+  }
+  else{
+    mnemonic = token2;
+  }
+  return mnemonic;
+}
+
+static int mnemToNumFacets( std::string mnemonic ){
```

Comment:
It does so automatically; should I add something to do so explicitly for readability?